app/src/main/java/ch/epfl/sweng/swenggolf/Config.java

```diff
@@ -103,4 +115,41 @@ public static ActivityCallback getActivityCallback() {
         return Config.activityCallback;
     }
 
+    /**
+     * Checks if the app is allowed to check the location. If not, requests it to Android.
+     *
+     * @param activity the current activity in which we ask for the permission
+     * @return whether we are authorized or not
+     */
+    public static boolean checkLocationPermission(Activity activity) {
```

Comment:
Isn't possible to check permissions and get them inside appLocation ? Or move this static method there since it is only necessary for the location ?

Reply:
Yes